reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -592,7 +592,9 @@ <h3 class="">Packages</h3>
               </ul>
             </td><td><em>Declared Licenses:</em>
               <dl>
-                <dd>EPL-1.0</dd>
+                <dd>
```

Comment:
Commit message: "See also" sentence is incomplete.

Reply:
Done.

Reply:
Hmm, unless it's a GitHub UI glitch, the message in https://github.com/heremaps/oss-review-toolkit/pull/1967/commits/4fb2335449db034c21ea0dc4d9faaa3e50e90180 still looks incomplete to me.

Reply:
Ah crap - it wasn't Github, sorry!